core/src/main/scala/com/banno/kafka/publish/PublishStream.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Jack Henry & Associates, Inc.®
```

Comment:
The year in these copyrights should technically speaking be 2024, but not a big deal.

Reply:
`headerCheck` fails with newer years, and I couldn't find an easy solution to having it pass with the new dates and I'm pretty sure copyright involves derivative works so I think we're good to just use the original date

Reply:
Some people think it must be the last date, and some people think that's fraud.  The most credible explanation I've heard is that it lasts 70 years past the last surviving author's death, so I've mostly decided to worry about other things.  It frees up my January 1sts, too.